problem_builder/mentoring.py

```diff
@@ -1101,9 +1115,7 @@ def try_again(self, data, suffix=''):
         }
 
     def author_preview_view(self, context):
-        context = context.copy() if context else {}
-        context['author_preview_view'] = True
-        return super(MentoringWithExplicitStepsBlock, self).author_preview_view(context)
+        return self.student_view(context)
```

Comment:
Isn't it the implicit behaviour to fall back to the student view?  (No need to change anything, it's fine to keep this explicit in any case.  I'm just curious.)